tests/README.md

```diff
@@ -0,0 +1,49 @@
+# OLS Console Tests
+The OLS console tests relies on upstream [openshift/console](https://github.com/openshift/console/tree/master) which provides fundamental configurations, views that we can reuse in openshift-tests-private web tests
+
+## Prerequisite
+1. [node.js](https://nodejs.org/) >= 18 & [yarn](https://yarnpkg.com/en/docs/install) >= 1.20
+2. upstream [openshift/console](https://github.com/openshift/console/tree/master) should be cloned locally
+3. upstream openshift/console dependencies need to be installed, for example we cloned openshift/console repo and save it to ~/OLS
+   - cd ~/OLS/console/frontend
+   - yarn install
```

Comment:
It actually looks like we are using `upstream` for just a couple of things, so I suggest we copy the code for those couple of things into our repo, then we don't need to install `upstream` locally and don't need to worry about the yarn dependency either.

I experimented locally and was able to run the tests without copying the console repo tests.

Reply:
I like the idea very much. It will simplify things on konflux side by a long margin so thank you for bringing it up. I did the changes and am now waiting on a cluster to make sure the tests still run as expected!